source/MaterialXRuntime/RtNode.cpp

```diff
@@ -38,7 +38,7 @@ RtPrim RtNode::createPrim(const RtToken& typeName, const RtToken& name, RtPrim p
     const PvtPrim* nodedef = PvtObject::ptr<PvtPrim>(master);
 
     const RtToken nodeName = (name == EMPTY_TOKEN ?
-        RtToken(nodedefSchema.getNode().str() + "1") : name);
```

Comment:
This is the small change. Instead of defaulting the name of an image_color3 to image1, it now defaults to image.